tests/test_viz.py

```diff
@@ -17,10 +17,12 @@
 def attributes_explorer(helpers):
     # Get JSONLD file path
     path_to_jsonld = helpers.get_data_path("example.model.jsonld")
+    path_to_graph = helpers.get_data_path("example.model.pickle")
 
```

Comment:
I don't quite understand whats going on here. Shouldn't you be creating a 2nd attributes explorer that uses the pickle file and then testing both the original and new object?

Reply:
I'm not seeing any new tests for the tangles tree object.